iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyLoading/SurveyItemLoading.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  SurveyItemLoading.swift
+//  Survey
+//
+//  Created by Bliss on 4/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SkeletonUI
+import SwiftUI
+
+struct SurveyItemLoading: View {
+
+    var body: some View {
+        GeometryReader { geo in
+            VStack(alignment: .leading, spacing: 16.0) {
```

Comment:
Can extract as common spacing values for reuse 🙏

Reply:
Added in https://github.com/blyscuit/kmm-swiftUI-IC/pull/59/commits/990883954214dc410b91eae965fe397e896990d8

Reply:
oh then you can also apply the new value here 😁
```suggestion
            VStack(alignment: .leading, spacing: .lineSpacing) {
```